src/protocols/rdp/.gitignore

```diff
@@ -1,38 +1,7 @@
 
-# Object code
```

Comment:
Why are all these `.gitignore` lines removed?

Reply:
I'm assuming they're inheriting from the root file, and didn't ever need to actually be there?

Reply:
For this particular file, the per-protocol `.gitignore` files were a holdover from the past where all of these were separate, independent projects. They are now covered by the top-level `.gitignore`. They were removed at the time that the top-level `.gitignore` needed to be updated for the original set of changes, since making the same changes within those files would be superfluous.

Several lines were removed from the top-level `.gitignore`, too - this is because of the use of `build-aux/` to house much of the build-time cruft.

Reply:
Makes sense.